src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -31,6 +32,19 @@ const isRouteEnabled = (route: string, chain?: ChainInfo) => {
   return !featureRoute || hasFeature(chain, featureRoute)
 }
 
+const NavItemLabel = ({ item }: { item: NavItem }) => {
+  const queueSize = useQueuedTxsLength()
+
+  if (item.href === AppRoutes.transactions.history) {
+    return <SidebarListItemCounter count={queueSize} />
+  }
+
+  if (item.href === AppRoutes.swap) {
```

Comment:
Since this is just a static property, I would move it to the navItems config. The queue is in this dynamic code only because we need to count the queue size.